be2-scala/README.md

```diff
@@ -46,7 +46,7 @@ ch_epfl_pop_Server {
 	http {
 		interface = "127.0.0.1"
 		port = "8000"
-		path = ""
+		path = "client"
```

Comment:
The final update is not completely reflected: it misses `client-path = "client"` and `server-path = "server"`

Reply:
Good catch, done !